authentication/exceptions.py

```diff
@@ -51,11 +51,25 @@ def __str__(self):
         return "Password is required to login"
 
 
-class RequirePasswordAndProfileException(PartialException):
-    """Authentication requires a password and profile"""
+class RequirePasswordAndAddressException(PartialException):
```

Comment:
Might be a little verbose, but `RequirePasswordAndPersonalInfoException` would be a more accurate name. This exception is raised if there is a missing password, address, OR first/last/full name